proxyvisor/proxyvisor.go

```diff
@@ -0,0 +1,21 @@
+package proxyvisor
+
+import (
+	"math/rand"
+	"time"
+)
+
+func Provision() (string, error) {
+	// Simulate proxyvisor whilst we wait for it to be released by returning a random 62 char string
+	return random(62), nil
+}
+
+func random(strlen int) string {
```

Comment:
Seems like it ought to be in helpers given it's a generic function. `random` is not a very descriptive name either.